upstreamed: https://github.com/lxc/lxcfs/pull/133

--- lxcfs-2.0.2/configure.ac~	2016-06-28 21:27:44.000000000 +0300
+++ lxcfs-2.0.2/configure.ac	2016-08-26 00:14:40.468048818 +0300
@@ -66,6 +66,7 @@
 	AC_CHECK_FILE(/etc/mandrakelinux-release, with_distro="openmandriva")
 	AC_CHECK_FILE(/etc/mandriva-release,with_distro="openmandriva")
 	AC_CHECK_FILE(/etc/pardus-release,with_distro="pardus")
+	AC_CHECK_FILE(/etc/pld-release,with_distro="pld")
 fi
 with_distro=`echo ${with_distro} | tr '[[:upper:]]' '[[:lower:]]'`
 
@@ -76,7 +77,7 @@
 	ubuntu)
 		distroconf=default.conf.ubuntu
 		;;
-	redhat|centos|fedora|oracle|oracleserver)
+	redhat|centos|fedora|oracle|oracleserver|pld)
 		distroconf=default.conf.libvirt
 		;;
 	*)
@@ -108,6 +109,9 @@
 			ubuntu)
 				init_script=upstart,systemd,sysvinit
 				;;
+			pld)
+				init_script=systemd,sysvinit
+				;;
 			slackware)
 				echo -n "Warning: bsd init job not yet implemented"
 				init_script=
